src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -2794,6 +2795,7 @@ namespace AirflowNetworkBalanceManager {
 
         // Validate adjacent temperature and Enthalpy control for an interior surface only
         for (int i = 1; i <= AirflowNetworkNumOfSurfaces; ++i) {
+            if (MultizoneSurfaceData(i).SurfNum == 0) continue;
```

Comment:
Is it necessary to check if SurfNum == 0? A previous error check was performed as below:
![Capture2](https://user-images.githubusercontent.com/5970656/93223296-e5482e00-f73d-11ea-92f1-44b945e5d0f5.PNG)

Reply:
Fatal error added here as suggested, and extra checks for SurfNum==0 have been removed.